### DISCUSSION TITLE
Machine Readable Files Schema Release Freeze Date?

### DISCUSSION THREAD

**Discussion by ghost** [CMSgov/price-transparency-guide#45] [Q&A] [answered]
Will there be an established freeze date on the schemas to allow organizations required to generate these files the ability to work from a specific release version of the schemas? 

It appears based on these discussions there are many ideas and corrections related to the schemas, but their is still an established due date of 01/01/22 for the files.

Organizations generating these files will need to have time based on a finalized/version of the schemas for a 01/01/22 release to plan the proper design, development and software quality management for the effort involved based on a repeatable monthly process.

**Comment by shaselton-usds** [comment, marked as answer]
Just for completeness sake -- starting to wrap up some old threads. Sorry for the notification on an old thread.

The finalization date for version 1.0 (in which compliance will be measured) was March 1st, 2022. The schema will continue to iterate beyond v 1.0 when new updates are introduced.

July 1st, 2022 will continue to be the implementation date.
